definitions/ext-access_point/summary_metrics.yml

```diff
@@ -0,0 +1,14 @@
+ipAddress:
+  title: IP Address
+  unit: STRING
+  tag:
+    key: device_ip
+
+model:
+  title: Model
+  unit: STRING
+  query:
+    select: latest(SysDescr)
```

Comment:
Same as the other one. It may be worth making this a tag.

Reply:
There are times when the SysDesc of a device can be several hundred characters in length. Without being able to predict this field, I moved away from that idea.